elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -0,0 +1,102 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+from typing import TypedDict, Tuple, List
+
+
+class HintsDict(TypedDict):
+    """A class with type signatures for the partial scores dict"""
+    html_hints: List[Tuple[int, str]]
+    submission_count: int
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    # Set submit count
+    if ('hidden-hints' in data['params']):
+        raise ValueError("Duplicate params key: 'hidden-hints'.")
+
+    hints_dict = data['params'].setdefault('hidden-hints', dict())
```

Comment:
the convention we have for variable names (even dictionary keys) is to use `snake_case`, so `hidden_hints`.